packages/expo-image-picker/ios/EXImagePicker/EXImagePicker.m

```diff
@@ -27,7 +27,7 @@ @interface EXImagePicker ()
 @property (nonatomic, strong) NSDictionary *customButtons;
 @property (nonatomic, weak) EXModuleRegistry *moduleRegistry;
 @property (nonatomic, weak) id<EXPermissionsInterface> permissionsModule;
-
+@property (nonatomic) BOOL restoreStatusBarVisibility;
```

Comment:
```suggestion
@property (nonatomic, assign) BOOL shouldRestoreStatusBarVisibility;
```

I know it's the default, but everywhere else we explicitly define `assign`, so let's define it here too

Reply:
I will change it, but it's make no sense IMHO 😅